### DISCUSSION TITLE
Redmatic mit Tradfri Shortcut Button

### DISCUSSION THREAD

**Discussion by Arni74** [rdmtc/RedMatic#567] [Q&A]
Hallo, ich habe Redmatic mit dem Tradfri on /off switch laufen, ohne Problem. ich kann mit "msg.payload" die Nachricht auslesen und weiter verwenden. jetzt wollte ich die Shortcut buttons verwenden. Sie wurden unter "Permit Join" gefunden und unter "devices" angezeigt. Soweit alles normal. nur wenn ich sie in Coverter einbinde und mit msg.payload auslesen will, passiert nichts. 

Redmatic 7.2.1
node-red-contrib-zigbee 0.20.3

**Comment by Termite254** [comment]
Hi, das Problem kenne ich. Ich konnte das lösen in dem ich bei dem Device im Node einmal den Config Haken gesetzt habe (Node zigbee Controller). Danach kam auch bei dem Shortcut Button was in der msg.payload an. Läuft seit über einem halben Jahr stabil mit mehreren Shortcut Buttons. Von meinem iPhone gesendetAm 19.12.2022 um 16:48 schrieb Arni74 ***@***.***>:﻿
Hallo, ich habe Redmatic mit dem Tradfri on /off switch laufen, ohne Problem. ich kann mit "msg.payload" die Nachricht auslesen und weiter verwenden. jetzt wollte ich die Shortcut buttons verwenden. Sie wurden unter "Permit Join" gefunden und unter "devices" angezeigt. Soweit alles normal. nur wenn ich sie in Coverter einbinde und mit msg.payload auslesen will, passiert nichts.

**Reply by Arni74** [reply]
Also ich habe jetzt in controller node bei dem punkt DevConfig bei config die Haken gesetzt. bis auf die Fehlermeldung : "configure failed 0x003c84fffe1ea3b6 (Taster 2) Bind 0x003c84fffe1ea3b6/1 genPowerCfg from '901' failed (AREQ - ZDO - bindRsp after 10000ms)" ist leider nichts passiert.

**Reply by Arni74** [reply]
Habe es jetzt mehrmals probiert, einen von meinen drei Tastern wurde eingebunden. Die anderen beiden nicht. Der funktionierende Tasdter ist unter Binds aufgeführt, die anderen beiden nicht. Habe versucht, sie händisch mit den selben werten einzutragen, kommt immer die Fehlermeldung : Bind 0x003c84fffe1ea3b6/1 genPowerCfg from '900' failed (AREQ - ZDO - bindRsp after 10000ms)

**Comment by hobbyquaker** [comment]
node-red-contrib-zigbee ist deprecated und auf RedMatic 9 nicht installierbar; Empfehlung ist zigbee2mqtt auf einem separaten Rechner (https://github.com/rdmtc/RedMatic/wiki/ZigBee). Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
